plugins/feeds/public/bambenek_osint_ipmaster.py

```diff
@@ -0,0 +1,90 @@
+import re
+from datetime import timedelta
+
+from core import Feed
+import logging
+
+from core.observables import Hostname, Ip
+
+
+class Bambenek_osint_ipmaster(Feed):
+    default_values = {
+        "frequency":
+            timedelta(minutes=60),
+        "name":
+            "Bambenek_osint_ipmaster",
+        "source":
+            "http://osint.bambenekconsulting.com/feeds/c2-masterlist.txt",
+        "description":
+            "Feed Provided By: John Bambenek of Bambenek Consulting, jcb@bambenekconsulting.com // http://bambenekconsulting.com",
```

Comment:
Please use the first line in the feed comments:

`Master Feed of known, active and non-sinkholed C&Cs indicators (Bambenek)`